### PR TITLE
Update the 'game over' message

### PR DESCRIPTION
**Pull request by J-D-Russell**
update the 'game over' message so people know how to play again.

### CODE REVIEW

**Comment by J-D-Russell** [comment]
comment1

**Review by J-D-Russell — COMMENTED** [review]
review comment

**Comment by Ranjiemin** [comment]
comment2

**Review by Ranjiemin — APPROVED** [review]
Approved

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (J-D-Russell) [inline index.html]: Suggestion included below:
```suggestion
x = [1,2,3]
```

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
 <script>
   // Initialization
```
Comment (Ranjiemin) [inline index.html:53]:
```suggestion
  // Initialization suggestion
```

Comment (J-D-Russell) [reply index.html:53]: not needed

**Pull request merged**